controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -4037,26 +4050,13 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
 
         const generatedAPIKey = ApiKeyGenerator.generate();
 
-        // We do not display subgraphs used for monographs in user accessible resources.
```

Comment:
Please add authorize checks for subgraph as well in all monograph rpcs which modify subgraph.